### PR TITLE
Create Gemfile to work locally

### LINKED ISSUE #3: Using Ruby gems and Gemfile ?
Hmm Are you not using any Jekyll plugins and Ruby Gems?

You can use Gems by setting up Ruby locally (rvm / rbenv) and then use bundler.
`bundle` command would install gems.

These gems can be then monitored by dependabot, which would update them automatically :) 
Plus some cool features of jekyll plugins can be used as well. (Worked like charm with minimal mistakes)

E.g. Gemfile https://github.com/oke-aditya/oke-aditya.github.io/blob/master/Gemfile

### PR DESCRIPTION
**Pull request by oke-aditya**
Closes #3

:smile: Should I annotate the PR so that you could learn how to use Jekyll like pro?

### CODE REVIEW

**Comment by oke-aditya** [comment]
Sadly there is no CI to check if this change would really "work". Something to learn as well :smiley:

**Comment by khushi-411** [comment]
Hey, @oke-aditya! I am very much thankful to you for such an outstanding contribution. ❤️

> Should I annotate the PR so that you could learn how to use Jekyll like pro?

Sure, why not? I would love to learn from you !!

> Although I can confirm locally that the site builds :)
Edit: For that you need to use a GH_TOKEN and pass it as secrets over GitHub and add a GitHub action. (Sound complicated right?)

Sounds great! Could you please provide more details about it?

Thanks a lot again!

**Comment by oke-aditya** [comment]
Why so much hurry. You could add CI and then merge. **Never merge a change which modifies config files without a CI**

File: Gemfile [hunk 1]
```diff
@@ -0,0 +1,27 @@
+source "https://rubygems.org"
+ruby "2.7.4"
+
+gem "jekyll", "~> 4.2.1"
```
Comment (oke-aditya) [inline Gemfile:4]: How did we get this Gemfile?
```
bundle init
```

Jekyll is a Gem! so are so many Jekyll plugins.
You can explore each of them and see what cool functionalities they can add.

File: Gemfile.lock [hunk 2]
```diff
@@ -0,0 +1,155 @@
+GEM
```
Comment (oke-aditya) [inline Gemfile.lock:1]: Gemfile.lock basically `lock` down versions of Gems. Don't worry this is auto-generated file
Run

```
bundle update
```

to generate this file.

File: _config.yml [hunk 3]
```diff
@@ -36,6 +36,3 @@ name: "Khushi Agrawal's Blog"
 
 redirects: yes
 permalink: /:title/
-
-gems:
```
Comment (oke-aditya) [inline _config.yml:40]: You earlier listed Gems in conifg. which isn't the best way of doing things.

File: .gitignore [hunk 4]
```diff
@@ -0,0 +1,6 @@
+_site
```
Comment (oke-aditya) [inline .gitignore:1]: As we build the site locally by running jekyll.

```
bundler exec jekyll serve
```

The site and metadata is built. We need to ignore these.

**Review by oke-aditya — COMMENTED** [review]
Hey @khushi-411 ! Glad to help you.

Here are the comments for PR. I forgot about this over the call.

Read more about jekyll, bundler, gems over the docs.

https://jekyllrb.com/docs/

You can either setup ruby by package manager or using rbenv

Setup Ruby with rbenv
https://github.com/rbenv/rbenv


If you have further doubts let me know!

**Pull request merged**